jar/jar_test.go

```diff
@@ -31,6 +31,8 @@ func TestParse(t *testing.T) {
 		filename string
 		wantBad  bool
 	}{
+		{"400mb.jar", false},
```

Comment:
Can you add some comments about how these files were generated?

Reply:
I added shell commands in `generate.sh` and comments for zipbombs that I used verbatim.

Reply:
thanks!